cpp/map_module/algorithm/map.cpp

```diff
@@ -0,0 +1,32 @@
+#include <list>
+#include <sstream>
+
+#include "map.hpp"
+
+void Map::FromPairs(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    const auto &list = arguments[0].ValueList();
+
+    mgp::Map pairsMap;
+
+    for (const auto inside_list : list) {
+      if (inside_list.ValueList().Size() != 2) {
```

Comment:
2 is magic number here, check following: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
Isolate that number somewhere